subsys/net/l2/wifi/wifi_nm.c

```diff
@@ -27,13 +27,15 @@ struct wifi_nm_instance *wifi_nm_get_instance_iface(struct net_if *iface)
 		return false;
```

Comment:
Not part of your changes but this is returning wrong value, we should return `NULL` here. Could you add a separate commit fixing this to your PR.

Reply:
Updated return to NULL.